examples/is31fl3731_text_example.py

```diff
@@ -1,19 +1,20 @@
 import board
 import busio
 import adafruit_framebuf
-import adafruit_is31fl3731
 
+# uncomment next line if you are using Feather CharlieWing LED 15 x 7
+# from adafruit_is31fl3731.charliewing import CharlieWing as Display
```

Comment:
I think this import needs an underscore like: `# from adafruit_is31fl3731.charlie_wing import CharlieWing as Display`

Reply:
Thank you, I fixed that.